src/sentry/api/endpoints/project_agnostic_rule_conditions.py

```diff
@@ -20,14 +20,18 @@ def info_extractor(rule_cls):
                 context["formFields"] = rule_cls.form_fields
             return context
 
-        # TODO(Jeff): Rename to `AlertDefaultsExperiment` on real experiment run
-        if experiments.get(org=organization, experiment_name="AlertDefaultsExperimentTmp") != 1:
-            return Response(status=status.HTTP_404_NOT_FOUND)
-
-        return Response(
-            [
-                info_extractor(rule_cls)
-                for rule_type, rule_cls in rules
-                if rule_type.startswith("condition/")
-            ]
+        experiment_variant = experiments.get(
+            org=organization, experiment_name="AlertDefaultsExperimentTmp"
```

Comment:
Should this still be Tmp?

Reply:
Yep, we are using the Tmp suffix because this isn't the actual rollout. We're pushing this to prod to test manually with test organizations to be sure before switching the experiment to actual rollout. Apologies for not documenting this